packages/volto/src/components/manage/Sidebar/__snapshots__/Sidebar.test.jsx.snap

```diff
@@ -59,11 +59,46 @@ Array [
         >
           Order
         </a>
+        <a
+          className="item"
+          onClick={[Function]}
+        >
+          Favorite
+        </a>
       </div>
       <div
         className="ui bottom attached segment active tab tab-wrapper"
         id="sidebar-metadata"
-      />
+      >
+        <button
```

Comment:
Why aren't you using SemanticUI components?

Reply:
Semantic UI is deprecated. See https://6.docs.plone.org/volto/upgrade-guide/index.html#semantic-ui.